### ISSUE TITLE
Better way to make value ranges

### ISSUE THREAD

**Issue by apoapostolov** [death-save/combat-utility-belt#138]
I am currently using these two conditions to replicate Bloodied (50%or less hp) and Beaten (25% or less hp)

![image](https://user-images.githubusercontent.com/4083812/80572890-b556f300-8a07-11ea-9c27-5fbc864e5a91.png)

After updating to 1.0.3 the logic for changing the value while in the same condition works well, but when changing Bloodied to Beaten and from Beaten to Bloodied by adding 1 or subtracting 1 from the hp, creates fringe cases where the condition

What should happen from crossing from Beaten to Bloodied by gaining 1 hp
- Beaten is removed
- Bloodied is applied

What happens actually
- Beaten is overlapped by Bloodied
- The condition is removed

**Comment by apoapostolov** [comment]
A similar situation with the previous bug that was solved in 1.0.3 is created today when the ranges apply for two conditions. While both Bloodied and Beaten conditions apply in the range 1% to 25%, healing 1 point of damage causes the conditions to trigger between Bloodied and Beaten and vice versa.

**Comment by apoapostolov** [comment]
I think this can be easily resolved by allowing to have range as a "activate" trigger, i.e. 1% to 25% for Beaten and 25%+1 to 50% for Bloodied.

**Comment by eclarke12** [comment]
Yes having a range was something I wanted to pursue but ran out of time. I'm going to adjust this issue to track the addition of that feature.
